cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/ldap/LdapUserPersistence.java

```diff
@@ -578,7 +670,30 @@ public void resetPassword(HttpPrincipal userID, String newPassword)
             conns.releaseConnections();
         }
     }
-
+    
+    private boolean checkIfGroupExists(final Group group, final LdapGroupDAO groupDAO) {
+    	boolean groupExists = false;
+        try {
+            Group tempGroup = groupDAO.getGroup(group.getID().getName(), false);
+            if (tempGroup != null) {
+                groupExists = true;
+            }
+        } catch (GroupNotFoundException ex) {
```

Comment:
Maybe in an early iteration of the code changes, LdapGroupDAO would return a null. I don't see how it can return null now. I shall remove the conditional check.